### DISCUSSION TITLE
¿Alguien sin equipo? [Grupo completo]

### DISCUSSION THREAD

**Discussion by ElizabethLeonPerez** [Launch-X-Latam/MisionFrontEnd#590] [Team Posts]
Hola! ¿Cómo están? 

¿Alguien sin equipo? Me estoy poniendo al día con las prácticas así que quiero empezar a buscar un equipo para el Hackaton. Mi mail de Launch es [lx06487@innovaccion.mx](mailto:lx06487@innovaccion.mx) 

Y si alguien tambien quiere formar grupo para ayudarnos con las prácticas y darnos feedback entre sí, re bievenido.

Feliz año nuevo a todos!!

**Comment by RobertRamos93** [comment]
Que tal, Elizabeth.

Buen día, también busco equipo.

lx02951@innovaccion.mx es mi correo; de igual manera te contacto por ese medio.

Feliz año.

**Comment by LuisCarlosJaramillo** [comment]
Voy muy atrasado, estaba sacando otros dos programas y no hice ni la ruta
de frontend, ahora ya tengo tiempo pero creo que es muy tarde para ponerme
al día con Launch-x

**Comment by mrivh** [comment]
¡Hola! Yo estoy sin equipo, te mando mensaje para hacer un grupo.

**Comment by ElizabethLeonPerez** [comment]
> Voy muy atrasado, estaba sacando otros dos programas y no hice ni la ruta
> de frontend, ahora ya tengo tiempo pero creo que es muy tarde para ponerme
> al día con Launch-x
> 
> El mié., 28 de dic. de 2022, 10:53, Roberto Ramos ***@***.***>
> escribió:
> …
> 
>  Que tal, Elizabeth.
> 
>  Buen día, también busco equipo.
> 
>  ***@***.*** es mi correo; de igual manera te contacto por ese
>  medio.
> 
>  Feliz año.
> 
>  —
>  Reply to this email directly, view it on GitHub
>  <https://github.com/orgs/Launch-X-Latam/teams/mision-backend/discussions/131/comments/1>,
>  or unsubscribe
>  <https://github.com/notifications/unsubscribe-auth/ATSEAVJY3TC7X7PSU5PV2SLWPRPBJANCNFSM6AAAAAATLFXZHY>
>  .
>  You are receiving this because you are subscribed to this thread.Message
>  ID: ***@***.***>

Yo también vengo atrasada y me estoy poniendo al día. Mientras le pongas ganas, sumate! Entre todos nos podemos ayudar

**Comment by mrivh** [comment]
Te mandé mensaje en Teams Elizabeth :)) mi correo ahí es lx04194@innovaccion.mx

**Comment by SNOOPYFC** [comment]
hola, yo quisera unirme
me aceptan?
